drivers/counter/counter_mspm0gxxxx.c

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+//
+// SPDX-License-Identifier: LicenseRef-BnOProprietary
```

Comment:
Not sure we can enforce this license on a public repo - also it goes against the zephyr licenses afaik.
@DmtKats you were in charge of this license afaik, do you have a saying here?

Reply:
switched to Apache 2.0 license